contrib/kind-common

```diff
@@ -387,9 +387,11 @@ install_kubevirt_ipam_controller() {
 }
 
 install_multus() {
-  echo "Installing multus-cni daemonset ..."
-  multus_manifest="https://raw.githubusercontent.com/k8snetworkplumbingwg/multus-cni/master/deployments/multus-daemonset.yml"
-  run_kubectl apply -f "$multus_manifest"
+  local version="v4.1.0"
+  echo "Installing multus-cni $version daemonset ..."
+  wget -qO- "https://raw.githubusercontent.com/k8snetworkplumbingwg/multus-cni/${version}/deployments/multus-daemonset.yml" |\
+    sed -e "s|multus-cni:snapshot|multus-cni:v4.1.0|g" |\
```

Comment:
is this supposed to temporary, or we'll want to keep tracking pinned releases ?

Reply:
Not sure. I think it is good that its pinned unless we have a reason not to.

Reply:
` sed -e "s|multus-cni:snapshot|multus-cni:v4.1.0|g"`
 
that's no good, fixing...